src/rpc/budget.cpp

```diff
@@ -712,14 +712,15 @@ UniValue mnfinalbudgetsuggest(const JSONRPCRequest& request)
     if (request.fHelp || !request.params.empty())
         throw std::runtime_error(
                 "mnfinalbudgetsuggest\n"
-                "\nTry to submit a budget finalization\n");
+                "\nTry to submit a budget finalization\n"
+                "returns the budget hash if it was broadcasted sucessfully");
 
     if (!Params().IsRegTestNet()) {
         throw std::runtime_error("mnfinalbudgetsuggest enabled only in regtest");
     }
 
-    budget.SubmitFinalBudget();
-    return NullUniValue;
+    uint256 budgetHash = budget.SubmitFinalBudget();
```

Comment:
(4ff56e7beec709fb891ca54f83568b04662d60e5)
nit:
```suggestion
    const uint256& budgetHash = budget.SubmitFinalBudget();
```